sprokit/processes/vxl/kw_archive_writer_process.h

```diff
@@ -1,5 +1,5 @@
 /*ckwg +29
- * Copyright 2014-2016 by Kitware, Inc.
+ * Copyright 2014-2017 by Kitware, Inc.
  * All rights reserved.
```

Comment:
Should this be 2018?

Reply:
These changes were technically done in 2017, they were cherry-picked off a large branch for ease.